### DISCUSSION TITLE
bticino shutter control wtih percentage

### DISCUSSION THREAD

**Discussion by fwollen** [FredBlo/node-red-contrib-myhome-bticino-v2#26] [Q&A]
Hi, is there any possibility to controll devices such as F401 or K4672M2S which have "presets" open and closed position? My intentions is to use percentage commands to open or close the shutters.
Thank you

**Comment by FredBlo** [comment]
Hi @fwollen

I personally do not have any shutter in my installation, which makes testing not really possible for me.
I actually kept the shutter part implementation as it worked in its initial version (i.e. supporting UP / DOWN / CLOSE) and improved all BUS connection concepts, stability,.. as I did for other nodes I could test.

As far as I read in OpenWebNet documentation, it should be working using commands such as :
`*#2*<where>*#11#<shutterPriority>*<shutterLevel>##`

`<where>` and `<shutterLevel>` params are quite logical but the `<shutterPriority>`  a bit less...

Have you tried sending commands on the BUS (through the 'MH Inject' node) using this structure, as for example:
`*#2*16*#11#0*50##`
(which should move shutter 1.6 to 50%)
